website/docs/r/maintenance_window.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: maintenance_window"
+sidebar_current: "docs-mongodbatlas-resource-maintenance_window"
+description: |-
+    Provides an Maintenance Window resource.
+---
+
+# mongodbatlas_maintenance_window
+
+`mongodbatlas_maintenance_window` provides a resource to take a scheduled maintenance event for a project up to two times. 
```

Comment:
Change to:
provides a resource to schedule maintenance window for your MongoDB Atlas Project,   defer a scheduled maintenance up to two times, and to start an available maintenance ASAP.

Reply:
Thank you!